### ISSUE TITLE
two_factor_requirement Warning

### ISSUE THREAD

**Issue by heurtematte** [eclipse-csi/otterdog#339]
I get this warning with otterdog (import|apply) with latest otterdog commit https://github.com/eclipse-csi/otterdog/commit/915d2a6686becde2796c63cf0f91a1e241241ac3:

```shell
╷
│ Warning: failed to retrieve setting 'two_factor_requirement' via web ui:
│
│    Page.eval_on_selector: Failed to find element matching selector "input[type="checkbox"][name="two_factor_requirement"]"
╵
```

**Comment by netomi** [comment]
Looks like this checkbox has been renamed to:

![Image](https://github.com/user-attachments/assets/98d51b14-deea-4222-9b93-29a6ecb83dca)

Interestingly, there is now an additional option:

![Image](https://github.com/user-attachments/assets/3ff89baa-049c-4014-a085-0a218e6b5a85)

Only allow secure 2FA methods

**Comment by netomi** [comment]
will fix this asap

**Comment by netomi** [comment]
Fixed by caa7db76afad97fd6dfecd0a88dc2a68585d46ce.
